_posts/2020-05-20-gsoc-students.md

```diff
@@ -0,0 +1,131 @@
+style="float: left; width: 110px; height: 110px; border-radius: 20px; border: 15px solid white" />
+
+The aim of the [RDKit interoperability
+project](https://summerofcode.withgoogle.com/projects/#6750913248624640)
+is to give MDAnalysis the ability to use [RDKit][]’s `Chem.Mol`
+structure as an input to an MDAnalysis `Universe`, but also to
+convert a `Universe` or `AtomGroup` to an RDKit `molecule`. RDKit is
+one of the most complete and one of the most commonly used
+chemoinformatics package, yet it lacks file readers for formats
+typically encountered in MD simulations. @cbouy will implement in
+MDAnalysis the ability to switch back and forth between a `Universe`
+and an RDKit `molecule` to perform typical chemoinformatics
+calculations and so add a lot of value to both packages.
+
+Cédric is a PhD student in molecular modelling at Université Côte
+D'Azur, France. His research aims to decipher the molecular basis
+of chemosensory perception (smell and taste) using computational
+tools. His day-to-day work includes; modelling bitter taste receptors,
+building machine-learning models to search for molecules with
+interesting olfactive or sapid properties, maintaining the website
+of the Global Consortium of Chemosensory Researchers, and a bit of
+teaching. In his free time he enjoys cooking and playing video games.
+Cédric can be found on github as [@cbouy][cbouy-github] and on twitter
+as [@cedricbouysset][cbouy-twitter].
+
+Cédric will describe his progress in [his blog][cbouy-blog].
+
+
+## Yuxuan Zhuang: Serialize Universes for parallel
+
+<img
+src="https://i0.wp.com/www.biophysics.se/wp-content/uploads/2018/06/IMG_4767.jpg"
+title="Yuxuan Zhuang" alt="Yuxuan Zhuang"
+style="float: left; width: 110px; height: 110px; border-radius: 20px; border: 15px solid white" />
+
+As we approach the exascale barrier, researchers are handling
+increasingly large volumes of molecular dynamics (MD) data. Whilst
+MDAnalysis is a flexible and relatively fast framework for complex
+analysis tasks in MD simulations, implementing a parallel computing
+framework would play a pivotal role in accelerating the time to
+solution for such large datasets. To achieve a flawless
+implementation of parallelism, @yuxuanzhuang will implement
+[serialization support for
+`Universe`](https://summerofcode.withgoogle.com/projects/#5812065073102848),
+the core of MDAnalysis. Furthermore, he will adapt this new
+serialization functionality to accelerate MDAnalysis' analysis modules
+using distributed computing frameworks, e.g. Dask, multiprocessing, or
+MPI.
+
+Yuxuan is a PhD student at Stockholm University. He mainly works on
+understanding pentameric ligand-gated ion channels from MD simulations.
+His daily workflow involves setting up and running simulations,
+on lab clusters or HPC centers, and performing various analyses on the
+MD trajectories in his jupyter notebook. Yuxuan can be found on github
+as [@yuxuanzhuang][yuxuanzhuang-github].
```

Comment:
@IAlibay you don't need to explicitly link the GitHub – we have an extension that automatically expands the @-handle.